### PR TITLE
[CI] Move the signature to ESRP

### PR DESCRIPTION
**Pull request by mandel-macaque**
Mov the signature of the binaries to ESRP.

### CODE REVIEW

**Comment by mandel-macaque** [comment]
Build example: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=5452017&view=results

**Review by dalexsoto — APPROVED** [review]
Wow, it is happening!

File: tools/devops/automation/templates/build/sign-and-notarized.yml [hunk 1]
```diff
@@ -134,23 +134,41 @@ steps:
...
-  timeoutInMinutes: 90
+  timeoutInMinutes: 180
   condition: ${{ parameters.condition }}
 
+- task: ArchiveFiles@1
+  displayName: 'Archive ESPP temp'
+  inputs:
+    rootFolder: '$(System.DefaultWorkingDirectory)/esrp'
+    includeRootFolder: false
+    archiveFile: '$(Build.ArtifactStagingDirectory)/erp.zip'
```
Comment (tj-devel709) [inline tools/devops/automation/templates/build/sign-and-notarized.yml:159]: Just confirming that this shouldn't be esrp.zip?

File: tools/devops/automation/templates/build/sign-and-notarized.yml [hunk 2]
```diff
@@ -134,23 +134,41 @@ steps:
...
-    TEAM_ID: $(TeamID)
-    KEYCHAIN: $(SigningKeychain)
   name: notarize
   displayName: 'Signing Release Build'
-  timeoutInMinutes: 90
+  timeoutInMinutes: 180
   condition: ${{ parameters.condition }}
 
+- task: ArchiveFiles@1
+  displayName: 'Archive ESPP temp'
```
Comment (tj-devel709) [inline tools/devops/automation/templates/build/sign-and-notarized.yml:155]:
```suggestion
  displayName: 'Archive ESRP temp'
``` maybe?

File: tools/devops/automation/templates/build/sign-and-notarized.yml [hunk 3]
```diff
@@ -134,23 +134,41 @@ steps:
...
+    includeRootFolder: false
+    archiveFile: '$(Build.ArtifactStagingDirectory)/erp.zip'
+  continueOnError: true
+  condition: and(${{ parameters.condition }}, eq(variables['System.debug'], 'true'))
+
+# Upload the esrp temp dir
+- task: PublishPipelineArtifact@1
+  displayName: 'Publish Artifact: ESRP temp'
+  inputs:
+    targetPath: '$(Build.ArtifactStagingDirectory)/erp.zip'
```
Comment (tj-devel709) [inline tools/devops/automation/templates/build/sign-and-notarized.yml:167]: Just confirming that this one as well shouldn't be esrp.zip?

File: tools/devops/automation/templates/build/sign-and-notarized.yml (hunk 1 above)
Comment (mandel-macaque) [reply tools/devops/automation/templates/build/sign-and-notarized.yml:159]: sure.

File: tools/devops/automation/templates/build/sign-and-notarized.yml (hunk 3 above)
Comment (mandel-macaque) [reply tools/devops/automation/templates/build/sign-and-notarized.yml:167]: same.

File: tools/devops/automation/templates/build/sign-and-notarized.yml (hunk 2 above)
Comment (mandel-macaque) [reply tools/devops/automation/templates/build/sign-and-notarized.yml:155]:
```suggestion
  displayName: 'Archive ESRP temp'
```

Comment (mandel-macaque) [reply tools/devops/automation/templates/build/sign-and-notarized.yml:155]:
```suggestion
  displayName: 'Archive ESRP temp'
```

File: tools/devops/automation/templates/build/sign-and-notarized.yml (hunk 1 above)
Comment (mandel-macaque) [reply tools/devops/automation/templates/build/sign-and-notarized.yml:159]:
```suggestion
    archiveFile: '$(Build.ArtifactStagingDirectory)/esrp.zip'
```

File: tools/devops/automation/templates/build/sign-and-notarized.yml (hunk 3 above)
Comment (mandel-macaque) [reply tools/devops/automation/templates/build/sign-and-notarized.yml:167]:
```suggestion
    targetPath: '$(Build.ArtifactStagingDirectory)/esrp.zip'
```

**Review by rolfbjarne — APPROVED** [review]

**Comment by mandel-macaque** [comment]
Merging. Build was signed and successful: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=5454860&view=results

**Pull request merged**